### PR TITLE
Added README for the user account module

### PR DESCRIPTION
**Pull request by nicolasbrossard**
As the title says, added file `modules/user_accounts/README.md`

### CODE REVIEW

File: modules/user_accounts/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# User Accounts
+
+## Purpose
+
+The user accounts module is used to manage the user accounts on the LORIS 
+website. It provides functions to add new users, edit a user account and search
```
[the remark this answers is not in the document]
Comment (PapillonMcGill) [reply modules/user_accounts/README.md]: I second that

File: modules/user_accounts/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Intended Users
+
+This module is typically used by site coordinators to manage the access rights to the LORIS
+system at their site. Other users might include the study coordinator and the LORIS system 
+administrator.
+
+## Scope
+
+This module displays information about the user accounts on the LORIS website.
```
Comment (PapillonMcGill) [inline modules/user_accounts/README.md]: same as previous. (Loris website)

File: modules/user_accounts/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nicolasbrossard) [reply modules/user_accounts/README.md]: Done

File: modules/user_accounts/README.md (hunk 2 above)
Comment (nicolasbrossard) [reply modules/user_accounts/README.md]: Done also

File: modules/user_accounts/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+## Configurations
+
+The user accounts module has the following configuration settings that affect its usage:
+
+title
+ - title of the study that will appear in the text of the email sent to a user 
+and/or his/her supervisor when modifying or creating a user account. 
+
+url
+ - url of the LORIS website that will appear in the text of the email sent to a
```
Comment (PapillonMcGill) [inline modules/user_accounts/README.md]: Loris website -> Loris app

**Pull request merged**